code/__DEFINES/components.dm

```diff
@@ -11,7 +11,7 @@
 #define ELEMENT_DETACH		(1 << 0)
 
 // How multiple components of the exact same type are handled in the same datum
-
+#define COMSIG_ATOM_BUMPED "atom_bumped"

```

Comment:
could've just cherry-picked the PR from /tg/

Reply:
probably